scripts/connect_clouddb_proxy.py

```diff
@@ -41,10 +47,14 @@ def main(instance_id, local_port, verbose=False):
 
     # Get the command line option
     parser = argparse.ArgumentParser(description="Connect to a google CloudSQL via a local proxy")
-    parser.add_argument('-d', '--database', default=None, required=True,
-                        help="Database, currently 'meta' or 'tess' from the config.")
-    parser.add_argument('-v', '--verbose', action='store_true', default=False,
-                        help="Print results to stdout")
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument('--from_config', default=True, action='store_true',
+                       help="Connect to all instances listed in the config file, default True.")
+    group.add_argument('--database', default=None,
+                       help="Connect to a specific database listed in the config.")
+    parser.add_argument('--key_file', default=None, help="JSON service account key location.")
```

Comment:
Maybe we should settle on a default location, perhaps baked into pocs.yaml, such as `$HOME/.pocs_secrets/cloud_sql_proxy.key`.

Reply:
Yes, this is described here: https://github.com/panoptes/POCS/tree/develop/pocs/utils/google#service-account-key